recipe/meta.yaml

```diff
@@ -1,9 +1,9 @@
 {% set name = "sasview" %}
-{% set version = "v5.0_beta.2_8_g390494d7c" %}  # git describe --tags --dirty --always | sed 's/-/_/g'
```

Comment:
@CJ-Wright, I think it was already discussed that the package should have this version until it's officially released by the code maintainers. Can I just ignore this PR?

Reply:
yes you can ignore it

Reply:
Thank you!